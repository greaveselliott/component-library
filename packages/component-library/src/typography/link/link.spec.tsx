import React from 'react';
import { shallow } from 'enzyme';

import Link from './link.component';

test('Heading one should render correctly', () =>
  expect(shallow(<Link>abc</Link>)).toMatchSnapshot());
