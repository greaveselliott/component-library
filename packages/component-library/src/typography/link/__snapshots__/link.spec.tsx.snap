// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading one should render correctly 1`] = `
<a
  className="link"
>
  abc
</a>
`;
